pkg/common/common.go

```diff
@@ -64,3 +65,20 @@ func GetNamespaceLabels(namespaceObj *v1.Namespace, logger logr.Logger) map[stri
 	}
 	return namespaceUnstructured.GetLabels()
 }
+
+// SplitGVK - from GVK
+func SplitGVK(str, sep string) []string {
+	return strings.Split(str, sep)
```

Comment:
Since it's only 1 line method, do we really need to write a separate method?

Reply:
okay. I will marge.